mapApp/static/bootstrap/css/bootstrap-theme.min.css

```diff
@@ -1,7 +1,8 @@
+@import url('https://fonts.googleapis.com/css2?family=Carrois+Gothic+SC&display=swap');
```

Comment:
Interesting that you had to import the font into the bootstrap css

Reply:
missed this one- is there another location it should be imported to?